doc/sphinx/source/quickstart/installation.rst

```diff
@@ -393,4 +393,12 @@ confirm that no errors are reported:
 
 .. code-block:: bash
 
-    python setup.py test
+    pytest -m "not installation"
+
+or if you want to run the full test suite remove the `-m "not installation"` flag;
+also if you want to run the tests on multiple threads, making the run faster, use
+the `-n N` flag where N is the number of available threads e.g:
+
+.. code-block:: bash
+
+    pytest -n 4
```

Comment:
The best number of threads is automatically detected, so there should be no need to change this unless your machine reports a wrong number of CPUs (i.e. CircleCI) or you're on some shared machine and do not want to hog the entire machine

Reply:
On a jasmin shared node it defaults to 1 and you spend an eternity for the tests to finish running, so that's why I added the mention